src/common/ceph_context.cc

```diff
@@ -254,9 +254,11 @@ class CephContextObs : public md_config_obs_t {
 	conf->enable_experimental_unrecoverable_data_corrupting_features,
 	cct->_experimental_features);
       ceph_spin_unlock(&cct->_feature_lock);
+#if defined(NDEBUG)
```

Comment:
@tchaikov 
So we 100% agree. 
- it is for the end users
- it is NonDebug

So if I want to suppress this during developement, because it does not conviene any information to developers other than logfile pollution, it think it needs to be included in a:
```
#ifdef NDEBUG
#endif
```